angrmanagement/ui/main_window.py

```diff
@@ -153,9 +162,10 @@ def _init_toolbars(self):
     def _init_menus(self):
         fileMenu = FileMenu(self)
         analyzeMenu = AnalyzeMenu(self)
+        helpMenu = HelpMenu(self)
         self.menuBar().addMenu(fileMenu.qmenu())
         self.menuBar().addMenu(analyzeMenu.qmenu())
-
+        self.menuBar().addMenu(helpMenu.qmenu())
     #
```

Comment:
Do not remove the original blank line. There should be a blank line between two adjacent methods.